src/raygun.rum/vendor/web-vitals-polyfills.vendor.js

```diff
@@ -1,5 +1,9 @@
 (function () {
 
+    if(typeof document === 'undefined') {
```

Comment:
A comment explaining this guard (and the one below) would be good to have for future reference